commit fb09a07ac2dbe8d41c7e4e9f0bafa88de035a37c
Author: Emmanuele Bassi <ebassi@gnome.org>
Date:   2013-03-05 00:46:45 +0000

    timeline: Use the right function pointer offset
    
    A copy and paste thinko: the ::stopped signal is using the
    ClutterTimelineClass.completed slot instead of the .stopped one,
    thus preventing sub-classes of ClutterTimeline from overriding the
    signal's default closure.

diff --git a/clutter/clutter-timeline.c b/clutter/clutter-timeline.c
index 135335e..d88447d 100644
--- clutter/clutter-timeline.c
+++ clutter/clutter-timeline.c
@@ -803,7 +803,7 @@ clutter_timeline_class_init (ClutterTimelineClass *klass)
     g_signal_new (I_("stopped"),
 		  G_TYPE_FROM_CLASS (object_class),
 		  G_SIGNAL_RUN_LAST,
-		  G_STRUCT_OFFSET (ClutterTimelineClass, completed),
+		  G_STRUCT_OFFSET (ClutterTimelineClass, stopped),
 		  NULL, NULL,
 		  _clutter_marshal_VOID__BOOLEAN,
 		  G_TYPE_NONE, 1,
